core/ipc/sink/aws-sqs/client/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>org.opennms.core.ipc.sink</groupId>
+    <artifactId>org.opennms.core.ipc.sink.aws.sqs</artifactId>
+    <version>23.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.opennms.core.ipc.sink.aws.sqs</groupId>
+  <artifactId>org.opennms.core.ipc.sink.aws.sqs.client</artifactId>
+  <name>OpenNMS :: Core :: IPC :: Sink :: AWS :: SQS :: Client</name>
+  <packaging>bundle</packaging>
+  <properties>
+    <skipITs>true</skipITs>
+  </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+            <Bundle-RequiredExecutionEnvironment>JavaSE-1.8</Bundle-RequiredExecutionEnvironment>
+            <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+            <Bundle-Version>${project.version}</Bundle-Version>
+            <Import-Package>
+              org.opennms.core.ipc.sink.camel.server;resolution:=optional,
```

Comment:
Do we actually need this import?

Reply:
Yes and no. 
The `Import-Package` statement is set to `resolution=optional` in order to not have it resolved, as this is used only when running inside OpenNMS (Client and server jars with Spring). 

However when running the client bundle in osgi, the import from `org.opennms.core.ipc.sink.camel.server` cannot be resolved, but the package is never used in this scenario anyways. 

In my opinion `!org.opennms.core.ipc.sink.camel.server` would be more fitted.

This is why I decided to use that pattern instead of `org.opennms.core.ipc.sink.camel.server;resolution:=optional`